### PR TITLE
Clean up package_ rules NNBD migration

### PR DESCRIPTION
**Pull request by parlough**
Removes uses of the explicit `!` modifiers in the `package_` lints.

### CODE REVIEW

**Review by pq — APPROVED** [review]
Looks great!

**Comment by pq** [comment]
I'm in the middle of a big set of migration changes and I'll merge these in; thanks!

Maybe hold off on any more until I land the next batch though?

Much  appreciated!

**Comment by parlough** [comment]
@pq Ah sorry I didn't realize. I'll hold off!

**Comment by pq** [comment]
No worries at all!  Thanks for the continued contributions!  💎

**Pull request merged**
